### PR TITLE
Bug/labels are cut in the bottom and text is shown behind the screen

### LINKED ISSUE #2059: Labels on Send/Request Transaction screens are cut in the bottom (Android only)
### Description

*Type*: Bug

*Summary*: bottom part of labels is cut (e.g "y" in Currency)

screenshot_20171009-192904

screenshot_20171009-192845

### Reproduction

- Open Status
- Wallet
- Tap on Send button to get Send Transaction screen and check the labels
- Tap on Request button to get Send Transaction screen and check the labels

### Additional Information
* Status version: develop
* Operating System: 
Real device Google Pixel XL, Android 8.0

### LINKED ISSUE #2087: Transaction screen: Wallet balance text is shown behind the screen (takes more space than screen has)
### Description

*Type*: Bug

*Summary*: On Send Transaction and Request Transaction screens (Wallet, Unsigned tx or tx initiated from chat/DApp), in field Wallet Wallet amount overlaps the field and goes further to the edge of the screen
img_2938

img_2942

#### Expected behavior
If amount can be fully shown then show as much as we can with … after amount and ETH, e.g “Main Wallet 12345678901.123456… ETH” 
#### Actual behavior
Amount does not fit the field: text overlaps edge of the field border and goes behind the screen

### Reproduction
Account should have enough ether, e.g. 10.123456789012345678
- Open Status
- Open Wallet 
- Tap on Send and check the content of the Wallet field

### Additional Information
* Status version: develop
* Operating System: Real device iPhone 5s

### PR DESCRIPTION
**Pull request by flexsurfer**
[comment]: # (Please replace ... with your information. Remove < and >)
[comment]: # (To auto-close issue on merge, please insert the related issue number after # i.e fixes #566)
fixes #2059
fixes #2087

sorry for such a huge PR

status: ready

### CODE REVIEW

**Review by yenda — APPROVED** [review]
TLDR;

**Review by rasom — APPROVED** [review]
